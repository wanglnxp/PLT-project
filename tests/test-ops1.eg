int main(){
  print("+-*/");
  print(1 + 2);
  print(1 - 2);
  print(1 * 2);
  print(10 / 2);
  print("==");
  print(1 == 2);
  print(1 == 1);
  print("!=");
  print(1 != 2);
  print(1 != 1);
  print("< > <= >=");
  print(1 > 2);
  print(2 > 1);
  print(1 < 2);
  print(2 < 1);
  print(1 <= 2);
  print(1 <= 1);
  print(2 <= 1);
  print(1 >= 2);
  print(1 >= 1);
  print(2 >= 1); 
  return 0;
}

