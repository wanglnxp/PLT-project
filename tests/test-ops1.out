+-*/
3
-1
2
5
==
false
true
!=
true
false
< > <= >=
false
true
true
false
true
true
false
false
true
true
